src/test/kotlin/uk/gov/justice/digital/hmpps/hmppsassessforearlyreleaseapi/entity/AssessmentTest.kt

```diff
@@ -256,25 +260,25 @@ class AssessmentTest {
   @Test
   fun `should get the current task based on the assessment state`() {
     val offender = anOffender()
-    assertThat(anAssessment(offender, NOT_STARTED).currentTask()).isEqualTo(ASSESS_ELIGIBILITY)
-    assertThat(anAssessment(offender, ELIGIBILITY_AND_SUITABILITY_IN_PROGRESS).currentTask()).isEqualTo(ASSESS_ELIGIBILITY)
-    assertThat(anAssessment(offender, ELIGIBLE_AND_SUITABLE).currentTask()).isEqualTo(ENTER_CURFEW_ADDRESS)
-    assertThat(anAssessment(offender, INELIGIBLE_OR_UNSUITABLE).currentTask()).isNull()
-    assertThat(anAssessment(offender, AWAITING_ADDRESS_AND_RISK_CHECKS).currentTask()).isEqualTo(CONSULT_THE_VLO_AND_POM)
-    assertThat(anAssessment(offender, AWAITING_ADDRESS_AND_RISK_CHECKS).copy(victimContactSchemeOptedIn = true).currentTask()).isEqualTo(CHECK_ADDRESSES_OR_COMMUNITY_ACCOMMODATION)
-    assertThat(anAssessment(offender, ADDRESS_AND_RISK_CHECKS_IN_PROGRESS).currentTask()).isEqualTo(RECORD_NON_DISCLOSABLE_INFORMATION)
-    assertThat(anAssessment(offender, ADDRESS_AND_RISK_CHECKS_IN_PROGRESS).copy(addressChecksComplete = true).currentTask()).isEqualTo(RECORD_NON_DISCLOSABLE_INFORMATION)
-    assertThat(anAssessment(offender, ADDRESS_UNSUITABLE).currentTask()).isNull()
-    assertThat(anAssessment(offender, AWAITING_PRE_DECISION_CHECKS).currentTask()).isEqualTo(REVIEW_APPLICATION_AND_SEND_FOR_DECISION)
-    assertThat(anAssessment(offender, AWAITING_DECISION).currentTask()).isEqualTo(CONFIRM_RELEASE)
-    assertThat(anAssessment(offender, AWAITING_REFUSAL).currentTask()).isEqualTo(CONFIRM_RELEASE)
-    assertThat(anAssessment(offender, REFUSED).currentTask()).isNull()
-    assertThat(anAssessment(offender, APPROVED).currentTask()).isEqualTo(Task.APPROVE_LICENCE)
-    assertThat(anAssessment(offender, AWAITING_PRE_RELEASE_CHECKS).currentTask()).isNull()
-    assertThat(anAssessment(offender, PASSED_PRE_RELEASE_CHECKS).currentTask()).isNull()
-    assertThat(anAssessment(offender, TIMED_OUT).currentTask()).isNull()
-    assertThat(anAssessment(offender, POSTPONED).currentTask()).isNull()
-    assertThat(anAssessment(offender, OPTED_OUT).currentTask()).isEqualTo(ENTER_CURFEW_ADDRESS)
-    assertThat(anAssessment(offender, RELEASED_ON_HDC).currentTask()).isNull()
+    assertThat(anAssessment(offender, hdced, crd, sentenceStartDate, NOT_STARTED).currentTask()).isEqualTo(ASSESS_ELIGIBILITY)
```

Comment:
mmh. is there some sensible we could default these too? They don't affect this test at all so ideally we'd not specify them

Reply:
Will do this in a follow up PR